ups/eupspkg.cfg.sh

```diff
@@ -13,4 +13,4 @@ export LUA_LIBS="-L$LUA_DIR/lib -llua"
 # Note: It is assumed that glib2 is installed system-wide and that
 # ./configure will detect it up via pkg-config.
 
-CONFIGURE_OPTIONS="--prefix=$PREFIX --with-lua --with-mysql=$MYSQL_DIR/bin/mysql_config"
+CONFIGURE_OPTIONS="--prefix=$PREFIX --with-lua --with-mysql=$MARIADB_DIR/bin/mysql_config"
```

Comment:
I may be mistaken, but `MARIADB_DIR` is the eups environment variable for the mariadb package, so I don't think this makes sense. Maybe we could do `${MARIADB_DIR:-${MYSQL_DIR}}`, so that mysqlproxy can build against either MySQL or MariaDB. Another option would be to have mysqlproxy-mariadb and mysqlproxy-mysql packages that depend on mariadb and mysql.

Finally, I don't know if eups has a way to say "I need either package set {X} or package set {Y}", so unless you are willing to have separate eups products for different backends, I guess we just have to mark both mysql and mariadb as optional dependencies and then enforce the presence of one or the other in our build scripts.